src/Admin/Ajax.php

```diff
@@ -310,4 +312,36 @@ public function payplug_init() {
 
 	}
 
+	/**
+	 * @return bool|null
+	 */
+
+	public function payplug_logout() {
```

Comment:
we have the exact same method in the payplugGateway, what do you think about put this in an helper?